content/en/real_user_monitoring/browser/setup/server/java_servlet.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Java Servlet
+beta: true
+code_lang: java
+type: multi-code-lang
+code_lang_weight: 7
+further_reading:
+- link: '/real_user_monitoring/browser/setup/server/'
+  tag: 'Documentation'
+  text: 'Browser Monitoring Auto-Instrumentation'
+---
+
+{{< callout url="https://forms.gle/RdVvDsmmzW21s2vd7" header="Join the Preview!">}}
+  RUM Auto-Instrumentation through Java servlet-based web servers is in Preview. 
+{{< /callout >}}
+
+## Overview
+
+RUM Auto-Instrumentation automatically adds RUM monitoring to your web application server. This means you don't need to modify your frontend code directly. Instead, you can start collecting RUM data by editing a configuration file. However, if you want to track specific user actions (custom actions) or add custom event details (event attributes), you still need to add some code to your application.
+
+## How it works
+
+The Java SDK works by monitoring your servlet API (which handles web requests and responses). It automatically checks each HTTP response your server sends. When it detects an HTML response, it looks for the `<head>` section and automatically adds the RUM JavaScript code there. The rest of your HTML content remains unchanged. This approach adds minimal overhead, so it works well even with streaming responses (where content is sent in real-time).
+
+## Prerequisites
+
+### Server requirements
+- Java web application server with Servlet API 3.0 or higher support:
+  - Jetty 7+
+  - Tomcat 7+
+  - GlassFish 3+ / Payara 4.1+
+  - JBoss AS 6+ / Wildfly 8+
+  - Undertow 1+
+  - WebLogic 12+
+  - WebSphere AS 8+ / Open Liberty 17+
+
+### SDK requirements
+- Java SDK installed using either:
+  - [Automatic instrumentation][1] (SSI)
+  - [Manual Java SDK installation][2]
+
+### Datadog configuration
+- RUM Application created in Datadog
+- Configuration values ready:
+  - `clientToken`
+  - `applicationId` 
+  - `remoteConfigurationId`
+
+## Troubleshooting
```

Comment:
It felt odd here to jump straight from prerequisites to troubleshooting - it made me wonder where the setup docs were! Is this something that kind of just automatically kicks in if you have the prerequisites all configured? Or should some of what's currently in the prerequisites section be considered setup? For example, I'm looking at the "Datadog configuration" section wondering if there should be more instructions there, but I'll defer to you there on what people need to know and when.

Reply:
Hi @janine-c I agree with you that it feels jarring. The configurations steps were intentionally left out because the product is in preview, but I think it might be worth adding a statement that acknowledges this in some way. Thanks for the callout!